### DISCUSSION TITLE
Multi user support

### DISCUSSION THREAD

**Discussion by DataHearth** [photoprism/photoprism#2135] [Ideas]
Hello everyone !

As you might already know, the multi-user support is a highly anticipated feature for lots of people (including me !). 
But as the craze grows up, related tickets (#98 #782) beginned to be flooded with questions/responses.

To keep those tickets clean and remove unwanted noises, we should discuss about this feature here where @lastzero and other maintainers can keep answering to your questions 😄 .

Tickets will be for feature updates and development ideas.

Cheers !

**Comment by Underknowledge** [comment]
Coming from the releases and seeing What is new? -> Auth: https://github.com/photoprism/photoprism/issues/782 I'm wondering if OIDC is already implemented. 
Fail to find something related in the user guide

**Reply by DataHearth** [reply]
For the latest stable release, it is not implemented. But I think there lots of work done if you look at the code base.

**Reply by DataHearth** [reply]
It needs UI integretation for sure and adjustements on back-end and/or DB I think.

**Comment by DataHearth** [comment]
@lastzero Hey there! I would like to contribute to those issues. Do you have info what is the leftover ?

**Reply by lastzero** [reply]
We will complete this once core functionality is in a good shape. Had to fix and improve a ton of things so we don't drag technical debt along. See Development Preview.

**Reply by DataHearth** [reply]
You mean the `Preview` inside the Github project ? Either way, I can also help you on that :).

**Reply by lastzero** [reply]
see roadmap, preview column: https://github.com/photoprism/photoprism/projects/5
